### PR TITLE
Add skeleton of MVC framework

### PR DESCRIPTION
**Pull request by KSIISKYI**

### CODE REVIEW

File: app/Core/Router.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        foreach($this->routes as $route => $params) {
+            if (preg_match($route, $url, $matches)) {
+                $this->params = $params;
+                return true;
+            }
+        }
+        return false;
+    }
+
+    function run()
```
Comment (Navalag) [inline app/Core/Router.php:33]: This method can be splitted into multiple. 
Basically, every time you have more than 2 indentation levels it's a sign to extract a new method.

For now, it's just a recommendation, no actions are required.

File: app/Core/Router.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            } else {
+                echo "Не найден контроллер: <b>$$controller_path</b>"; 
+            }
+        } else {
+            header("HTTP/1.0 404 Not Found");
+            require  APP_ROOT . '/public/html_exceptions/404.html';
+            die();
+        }
+    }
+}
```
Comment (Navalag) [inline app/Core/Router.php:67]: Always add an empty line at the end of file. Just a good tone.

File: public/js/colorpicker.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+(function(h,k,j){function r(a){if(h.event&&h.event.contentOverflow!==j)return{x:h.event.offsetX,y:h.event.offsetY};if(a.offsetX!==j&&a.offsetY!==j)return{x:a.offsetX,y:a.offsetY};var b=a.target.parentNode.parentNode;return{x:a.layerX-b.offsetLeft,y:a.layerY-b.offsetTop}}function e(a,b,c){a=k.createElementNS(v,a);for(var d in b)a.setAttribute(d,b[d]);if(Object.prototype.toString.call(c)!="[object Array]")c=[c];b=0;for(d=c[0]&&c.length||0;b<d;b++)a.appendChild(c[b]);return a}function l(a,b,c){var d,
```
Comment (Navalag) [inline public/js/colorpicker.js:1]: Are you sure you need this library?

**Review by Navalag — APPROVED** [review]
There are some remarks regarding formating and details. But overall nice implementation.
Approved 👍🏻

File: public/js/colorpicker.js (hunk 3 above)
Comment (KSIISKYI) [reply public/js/colorpicker.js:1]: hahaha, sorry, I took static files from the last project and forgot to delete them, but everything will be fixed in the new pull request.

**Pull request merged**